repos.yaml

```diff
@@ -1042,3 +1042,8 @@ repositories:
     maintainers:
       - name: tp-cts-kubesquad
       - email: tp-cts-kubesquad@comcast.com
+  - name: equinor-charts
+    url: https://equinor.github.io/equinor-charts/charts/
```

Comment:
This URL doesn't appear to be accessible. GH Pages might not be setup. When I try to reach https://equinor.github.io/equinor-charts/charts/index.yaml I get a 404 error.

Did you mean to use https://equinor.github.io/helm-charts/charts/ instead?

Reply:
You are correct! Sloppy copy-paste there on my part, sorry about that, and thanks for good QA!

I've fixed in latest commit and double-checked it's correct on my end.